packages/flutter/test/widgets/visibility_test.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
```

Comment:
why these extra imports? As far as I can tell this file only has formatting changes?